x/sds/types/msg.go

```diff
@@ -32,7 +34,7 @@ func NewMsgUpload(fileHash []byte, reporter, uploader sdk.AccAddress) MsgFileUpl
 func (msg MsgFileUpload) Route() string { return RouterKey }
 func (msg MsgFileUpload) Type() string  { return ConstFileUpload }
```

Comment:
We need to rename this to follow all other msg type name style

Reply:
FileUploadTx and SdsPrepayTx

Reply:
OK, doing this with later push